File: APP/singer_job/lib/modules/users/entities/artista.dart
class Artista {
  int id;
  String nome;
  String email;
  String senha;
  String endereco;
  String telefone;
  String genero;
  String tipoArte;
  String biografia;
  String pathImage;
  String portifolio;

  Artista(
      {required this.id,
      required this.nome,
      required this.email,
      required this.senha,
      required this.endereco,
      required this.telefone,
      required this.genero,
      required this.tipoArte,
      required this.biografia,
      required this.pathImage,
      required this.portifolio});
}

File: APP/backend/src/models/Artista.js
// const { DataTypes } = require('sequelize');
// const sequelize = require('../config/database');

// const Artista = sequelize.define('Artista', {
//     id_artista: { 
//         type: DataTypes.INTEGER, 
//         primaryKey: true, 
//         autoIncrement: true 
//     },
//     nome: { 
//         type: DataTypes.STRING, 
//         allowNull: false 
//     },
//     email: { 
//         type: DataTypes.STRING, 
//         allowNull: false, 
//         unique: true,
//         validate: {
//             isEmail: true // Adicionando validação de email
//         }
//     },
//     senha: { 
//         type: DataTypes.STRING, 
//         allowNull: false 
//     },
//     endereco: { 
//         type: DataTypes.STRING 
//     },
//     telefone: { 
//         type: DataTypes.STRING 
//     },
//     genero: { 
//         type: DataTypes.STRING 
//     },
//     tipo_arte: { 
//         type: DataTypes.STRING 
//     },
//     biografia: { 
//         type: DataTypes.TEXT 
//     },
//     portfolio: { 
//         type: DataTypes.TEXT 
//     },
// }, { 
//     timestamps: true 
// });

// module.exports = Artista;
